pkg/splunk/enterprise/clustermaster.go

```diff
@@ -301,3 +311,20 @@ func PushMasterAppsBundle(c splcommon.ControllerClient, cr *enterpriseApi.Cluste
 
 	return splunkClient.BundlePush(true)
 }
+
+// helper function to get the list of ClusterMaster types in the current namespace
+func getClusterMasterList(c splcommon.ControllerClient, cr splcommon.MetaObject, listOpts []client.ListOption) (int, error) {
+	scopedLog := log.WithName("getClusterMasterList").WithValues("name", cr.GetName(), "namespace", cr.GetNamespace())
```

Comment:
Yes initially I was using this API, but later when I changed the design a bit this was not being used. But I left it here since it feels like a useful API that can come in handy in the future.